src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.RAND.cs

```diff
@@ -9,16 +9,15 @@ internal static partial class Interop
 {
     internal static partial class Crypto
     {
-        internal static unsafe bool GetRandomBytes(byte* pbBuffer, int count)
+        internal static bool GetRandomBytes(ref byte pbBuffer, int count)
         {
-            Debug.Assert(pbBuffer != null);
```

Comment:
Checking `pbBuffer != null` dereferences pbBuffer and compares the value against null... which would be comparing a byte value against null, not the ref.